keadm/cmd/keadm/app/cmd/edge/batchjoin.go

```diff
@@ -0,0 +1,394 @@
+// Copyright 2024 WeiXingEsther
```

Comment:
```suggestion
// Copyright 2024 WeiXingEsther
```
```suggestion
// Copyright 2024 The KubeEdge Authors.
```

Reply:
> 

Thank you for your suggestions, I'll make changes.